runtime/src/bank/pyth/accumulator.rs

```diff
@@ -132,6 +146,7 @@ pub fn get_accumulator_keys() -> Vec<(
             "STAKE_CAPS_PARAMETERS_ADDR",
             Ok(*STAKE_CAPS_PARAMETERS_ADDR),
         ),
+        ("BATCH_PUBLISH_PID", Ok(*BATCH_PUBLISH_PID)),
```

Comment:
i recommend we rename this function and move it to mod.rs

Reply:
Done.